examples/seismic/model.py

```diff
@@ -430,11 +430,12 @@ def __init__(self, origin, spacing, shape, space_order, nbpml=20,
         # at the correct index
         origin_pml = tuple([dtype(o - s*nbpml) for o, s in zip(origin, spacing)])
         phydomain = PhysicalDomain(self.nbpml)
+        subdomains = subdomains + (phydomain, )
         shape_pml = np.array(shape) + 2 * self.nbpml
         # Physical extent is calculated per cell, so shape - 1
         extent = tuple(np.array(spacing) * (shape_pml - 1))
         self.grid = Grid(extent=extent, shape=shape_pml, origin=origin_pml, dtype=dtype,
-                         subdomains=phydomain)
+                         subdomains=subdomains)
```

Comment:
nice !

Reply:
> I'd drop the fd for the sake of generality

Dropped.

> the method should be private

I think you were linking to and older version of the code, this got changed to a `cached_property`. The method needs to be called from `coefficients.py` and `finite_difference.py` but if `coefficients='symbolic'` has not been declared it doesn't need to be created at all. I guess you're saying that accessing the public `cached_property` (or whatever is becomes) should internally call a private method? But does that make a difference?